.github/workflows/check_cirrus_cron.yml

```diff
@@ -62,24 +62,7 @@ jobs:
 
             - if: steps.cron.outputs.failures > 0
               shell: bash
-              # Must be inline, since context expressions are used.
-              # Ref: https://docs.github.com/en/free-pro-team@latest/actions/reference/context-and-expression-syntax-for-github-actions
-              run: |
-                set -eo pipefail
-                (
-                echo "Detected one or more Cirrus-CI cron-triggered jobs have failed recently:"
-                echo ""
-
-                while read -r NAME BID; do
-                    echo "Cron build '$NAME' Failed: https://cirrus-ci.com/build/$BID"
-                done < "$NAME_ID_FILEPATH"
-
-                echo ""
-                echo "# Source: ${{ github.workflow }} workflow on ${{ github.repository }}."
-                # Separate content from sendgrid.com automatic footer.
-                echo ""
-                echo ""
-                ) > ./artifacts/email_body.txt
+              run: './_podman/.github/actions/check_cirrus_cron/make_email_body'
```

Comment:
Does this script exists in Buildah?

Reply:
Nope, it's in podman.  It's also broken, so this PR should depend on https://github.com/containers/podman/pull/16414